kore/src/Kore/Attribute/Symbol.hs

```diff
@@ -182,8 +182,8 @@ instance Default Symbol where
     def = defaultSymbolAttributes
 
 -- | Is a symbol non-simplifiable?
-isNonSimplifiable :: StepperAttributes -> Bool
-isNonSimplifiable = do
+isConstructorLike :: StepperAttributes -> Bool
```

Comment:
The comments above and below should rename `non-simplifiable` to `constructor-like`.